src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -149,6 +147,24 @@ function EmojiPickerMenu({preferredLocale, onEmojiSelected, preferredSkinTone, t
         );
     };
 
+    /**
+     * Improves FlashList's recycling when there are different types of items
+     * @param {Object} item
+     * @returns {String}
+     */
+    const getItemType = (item) => {
```

Comment:
I've seen similar function in EmojiPickerMenu - could YOu share just one function?